packaging/macos/tests/run_tests.sh

```
@@ -0,0 +1,167 @@
+TEST_FAIL_COUNT=0
+TEST_OK_COUNT=0
+CUR_TEST_FILE=""
+CUR_TEST_NAME=""
+
+_fail() {
+  local msg="$1"
+  printf 'FAIL  %s :: %s\n  at: %s\n  %s\n' \
+    "${CUR_TEST_FILE}" "${CUR_TEST_NAME}" "${BASH_SOURCE[2]}:${BASH_LINENO[1]}" "${msg}" >&2
+  TEST_FAIL_COUNT=$((TEST_FAIL_COUNT + 1))
+  return 1
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)